finetune/finetune.py

```diff
@@ -77,10 +84,26 @@ def train_func(config: Dict[str, Any]):
                 offload_to_cpu=False, rank0_only=False
             ),
         )
+        deepspeed_plugin = None
+
+    elif accelerate_mode in ["GPU_DEEPSPEED"]:
+        fsdp_plugin = None
+        hf_ds_config = config["Training"]["deepspeed_config_file"]
```

Comment:
``` python
with open(config["Training"]["deepspeed_config_file"]) as f:
    hf_ds_config = yaml.full_load(f)
```
here `hf_ds_config` is a dict type object, and `DeepSpeedPlugin` can accept and process a dict type hf_ds_config.